crates/cast/tests/helpers/constants.rs

```diff
@@ -11,15 +11,15 @@ pub const CONTRACTS_DIR: &str = "tests/data/contracts";
 pub const MAP_CONTRACT_ADDRESS_V1: &str =
     "0x6a9b1c31fa25ca8f3cecb58142f3ec4d1f7611ed9520b4d3ae598d02d625e41";
 pub const MAP_CONTRACT_ADDRESS_V2: &str =
-    "0x49a028577be178d70912fd84bd6a9056ba831d8153e45c2aac02c0539f72b6e";
+    "0x3b96d821f8754564fd0b8a9ded6f549b675d7767bae1a5706f5095f81f5e357";
 pub const MAP_CLASS_HASH_V1: &str =
     "0x76e94149fc55e7ad9c5fe3b9af570970ae2cf51205f8452f39753e9497fe849";
 pub const MAP_CLASS_HASH_V2: &str =
-    "0x06eb3fda9eaa7d99bb7c9e72c8d874988a5b9269f3f866773889bc5ab4ec9079";
+    "0x5e8e8e76fcdc72f519ff2697fb5a774f8a029b45f2b61f9664298ab288e4bb1";
 pub const CONTRACT_WITH_CONSTRUCTOR_CLASS_HASH_V1: &str =
     "0x02bb3d35dba2984b3d0cd0901b4e7de5411daff6bff5e072060bcfadbbd257b1";
 pub const CONTRACT_WITH_CONSTRUCTOR_CLASS_HASH_V2: &str =
-    "0x022546fa1ab93bfcd5e8b386a8282ae7d2046e6076a06916608b2e44345b2765";
+    "0x01a8f37a7de9da1336190b6d34a961d6ef881239e85431335c33fa2b95926543";
```

Comment:
what scarb version was it compiled with?

Reply:
@war-in?

Reply:
I think it was 0.5.2

Reply:
any idea why hashes changed then? compiler/devnet/scarb didn't, right?

Reply:
I changed the names of the contract packages,I assumed that it's the reason